zh/restore-from-azblob-using-br.md

```diff
@@ -5,36 +5,58 @@ summary: 介绍如何使用 BR 恢复 Azure Blob Storage 上的备份数据。
 
 # 使用 BR 恢复 Azure Blob Storage 上的备份数据
```

Comment:
preview: https://pingcap-docs-website-scaffold-1hq.pages.dev/zh/tidb-in-kubernetes/dev/restore-from-azblob-using-br